src/sentry/api/endpoints/organization_details.py

```diff
@@ -158,7 +158,11 @@ def save(self):
                     key=option,
                     value=type_(self.init_data[key]),
                 )
-                changed_data[key] = self.init_data[key]
+                # TODO(kelly): This will not work if new ORG_OPTIONS are added and their
+                # default value evaluates as truthy, but this should work for now with the
+                # current ORG_OPTIONS (assumes ORG_OPTIONS are falsy)
```

Comment:
This assumption makes me a little nervous since it seems awfully easy to invalidate without noticing. I don't have any better ideas, though.